plug.py

```diff
@@ -401,6 +401,30 @@ def notify_capidata(
     return error
 
 
+def notify_capi_fleetcarrierdata(
+    data: companion.CAPIData
+) -> Optional[str]:
```

Comment:
```suggestion
) -> str | None:
```

Reply:
I was following the same pattern as the existing function `notify_capidata`.  Should I change the return type for that function too?  Extending that question further, the pattern `-> Optional[str]:` is used for many functions across the codebase, perhaps there should be a separate PR to review those.